docs/Developers/Newcomers_Guide.rst

```diff
@@ -259,6 +259,7 @@ more about it, have patience and it is explained on the next step), and all
 you can do is wait. The best thing you can do while at this step is review
 other people's PRs. Not only will this help the maintainers with the workload,
 but this is one of the three core steps towards becoming a full-norm coalaian.
+Never close a PR without mentioning a reason.
```

Comment:
So, should I edit this line to as what you're saying:
`Never close a PR unless you are told to do so.`